uber/config.py

```diff
@@ -412,3 +412,9 @@ def _is_intstr(s):
     c.VOLUNTEER_CHECKLIST = [url for step, url in _items]
 
 stripe.api_key = c.STRIPE_SECRET_KEY
+
+if c.KIOSK_CC_ENABLED:
+    c.DOOR_PAYMENT_METHOD_OPTS.append([c.STRIPE, "Pay with credit card now (faster)"])
+
+if c.GROUPS_ENABLED:
+    c.DOOR_PAYMENT_METHOD_OPTS.append([c.GROUP, "Taking an unassigned Group badge (group leader must be present)"])
```

Comment:
Hm, well, that poses a bit of a problem for me. I guess I will have to reverse the polarity on these and remove options instead of adding them, then.